crescent/ext/tasks/loop.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from datetime import timedelta as _timedelta
+from typing import Callable, Sequence, overload
+
+from crescent.ext.tasks.task import Task, TaskCallbackT, link_task
+from crescent.internal import MetaStruct
+
+__all__: Sequence[str] = ("loop", "Loop")
+
+
+class Loop(Task):
+    def __init__(self, callback: TaskCallbackT, timedelta: float) -> None:
+        self.timedelta = timedelta
+        self.first_loop = True
```

Comment:
This doesn't appear to ever be set to false, as far as I can tell. Either means that we don't need this, or that task loops are constantly running.

Reply:
Or that I'm blind.

Reply:
oh yeah, thats not needed now.

Reply:
wait it is needed. Look at `_call_next()`. It is set to False after `super()._call_next()` is called